seedlet/style-rtl.css

```diff
@@ -5,13 +5,12 @@ Author: Automattic
 Author URI: https://automattic.com/
 Description: Seedlet is a free WordPress theme. A two-column layout and classically elegant typography creates a refined site that gives your works and images space to breathe - and shine. Seedlet was built to be the perfect partner to the block editor, and supports all the latest blocks. Writing, audio, illustrations, photography, video - use Seedlet to engage and direct visitors' eyes, without your theme getting in the way. And the responsive design shifts naturally between desktop and mobile devices. Seedlet is a great option for professionals and creatives looking for a sophisticated vibe. Whether you're looking to create a blog or a robust site promoting your business, do with simplicity, style, and Seedlet.
 Requires at least: 4.9.6
-Tested up to: 5.4.1
-Requires PHP: 7.3
-Version: 1.0.3-wpcom
+Tested up to: 5.5
+Version: 1.0.6-wpcom
```

Comment:
It looks like these are left over from a previous commit that wasn't fully compiled?